_sell/fips201ep.md

```diff
@@ -89,24 +89,30 @@ Review the test procedures, choose one of the application packages, and submit t
 
 ### PACS Application Package for New Systems or for Updates to Previously Approved Systems
 
-All applicants, please complete the following steps: 
+All applicants, please complete the following steps:
 
 1. Review the - [Approved Product List Application Guidance Document (PDF, April 2022)]({{site.baseurl}}/docs/fips201ep-Application-guidance.pdf){:target="_blank"}{:rel="noopener noreferrer"} – Instructions for completeing the Approved Product List Application Form.
 2. Complete the [Approved Product List Application Form (Word, April 2022)]({{site.baseurl}}/docs/fips201ep-application-form.docx){:target="_blank"}{:rel="noopener noreferrer"} – Required for each solution submission, new or upgrade.
-3. Include the following  document when adding a new series, or adding new product to a prior listed series, signed by a C- or VP-level individual:
+3. Provide the equipment table from the Approved Product List Application as a separate file. [Equipment Table GSA PACS Application v0.1 (XLSX, February 2023)]({{site.baseurl}}/docs/equipment-table-gsa-pacs-application-v0.1.xlsx){:target="_blank"}{:rel="noopener noreferrer"}
```

Comment:
@claytonjbarnette can you update this link and document name to remove the version number? This will decrease the number of updates that are needed when the document is updated. Plus, helps with standardizing links between versions.

Reply:
@idmken 👍